src/components/hooks/useEnterpriseCustomer.js

```diff
@@ -0,0 +1,6 @@
+import { useEnterpriseLearner } from '../app/data';
+
+export default function useEnterpriseCustomer() {
```

Comment:
[inform] there are now two `function useEnterpriseCustomer()` defined. This one will be removed in my draft PR.